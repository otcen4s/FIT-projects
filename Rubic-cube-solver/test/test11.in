632
511
544
266 546 344 366
123 136 345 452
123 121 221 552
334
166
455