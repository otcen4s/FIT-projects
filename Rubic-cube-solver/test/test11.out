632
511
544
266 546 344 366
123 136 345 452
123 121 221 552
334
166
455

543
511
544
266 546 451 466
123 133 442 552
123 122 332 552
334
166
166

243
211
144
566 546 451 554
523 133 441 556
523 122 333 226
234
166
166

243
211
511
633 446 451 554
622 333 441 554
555 222 333 221
466
166
166

343
111
111
233 446 451 255
222 333 441 255
555 222 334 144
666
666
566

443
111
111
333 446 455 122
122 333 446 455
155 222 336 455
266
266
566

311
411
411
122 333 446 455
122 333 446 455
155 222 336 455
266
266
566

111
111
111
222 333 444 555
222 333 444 555
555 222 333 444
666
666
666

111
111
111
222 333 444 555
222 333 444 555
222 333 444 555
666
666
666

